internal/test/scorecard/tests.go

```diff
@@ -124,6 +183,7 @@ func CryostatRecordingTest(bundle *apimanifests.Bundle, namespace string, openSh
 		Password:        string(secret.Data["CRYOSTAT_RJMX_PASS"]),
 		MatchExpression: fmt.Sprintf("target.alias==\"%s\"", target.Alias),
 	}
+	defer cleanupCryostat(r, tr.Client, CryostatCRTestName, namespace)
```

Comment:
I think we can remove it? On line 149, we defer cleanup

https://github.com/cryostatio/cryostat-operator/blob/cfcbfc74fab65b78bf5fb63a8c5abe658d65fe6a/internal/test/scorecard/tests.go#L90